src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -0,0 +1,536 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import {
+  Alert,
+  Button,
+  Popover,
+  Spinner,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import {
+  HelpIcon,
+  MinusCircleIcon,
+  PlusCircleIcon,
+} from '@patternfly/react-icons';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import styles from '@patternfly/react-styles/css/components/Table/table';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+// @ts-ignore
```

Comment:
Ok, googled a bit and this can be removed. All that needs to be done for the error to disappear is to install `@types/uuid` as a dev dependency. The dependency needs to have the same version as uuid in the package.json file.